### ISSUE TITLE
🐛 Staging is not indexing analyses

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/roadmap#589]
## Describe the bug
Staging is not indexing newly published analyses

## Steps To Reproduce
1. Query the RDPC gateway api (https://api.rdpc.cancercollaboratory.org/graphql)
```
query {
  analyses(filter: { analysisId: "a3307730-f180-4043-b077-30f180c043c4"}) {
    analysisId
    analysisType
    analysisState
    studyId
    analysisVersion
    files {
      objectId
      name
    }
    inputForRuns{
      runId
      producedAnalyses{
        analysisId
        analysisType
        analysisState
        files{
          objectId
          md5Sum
          name
        }
        inputForRuns {
          runId
        }
      }
    }
  }
}
```

2. See that there are 2 published analyses

```
            "producedAnalyses": [
              {
                "analysisId": "2ab07a29-e008-4fd6-b07a-29e008bfd642", <--------------------LOOK HERE
                "analysisType": "sequencing_alignment",
                "analysisState": "PUBLISHED",
                "files": [
                  {
                    "objectId": "f98c4e70-eea7-5895-8d22-92edf7096348",
                    "md5Sum": "23124cb51114f1b0bfbd46db8c788e69",
                    "name": "OCCAMS-GB.DO50328.SA528818.wgs.20200930.aln.cram"
                  },
                  {
                    "objectId": "e014f882-4a86-581c-be19-8c069036d76d",
                    "md5Sum": "c3b2265821ac8130793eeeb0146fe6c5",
                    "name": "OCCAMS-GB.DO50328.SA528818.wgs.20200930.aln.cram.crai"
                  }
                ],
                "inputForRuns": []
              },
          {
            "runId": "wes-95060b4afa6f482f9722cddc1a0e1e66",
            "producedAnalyses": [
              {
                "analysisId": "6939ef36-4f0f-4a43-b9ef-364f0f0a4369",   <--------------------LOOK HERE
                "analysisType": "sequencing_alignment",
                "analysisState": "PUBLISHED",
                "files": [
                  {
                    "objectId": "add4a661-e558-5ff0-87bc-5b3d5eb1f451",
                    "md5Sum": "a57bc65ef120a7f64dddb681a0a51e7d",
                    "name": "OCCAMS-GB.DO50328.SA528818.wgs.20200402.aln.cram"
                  },
                  {
                    "objectId": "f66ace0d-025f-5536-ac84-9903dc4d2ec3",
                    "md5Sum": "9cd8f5ec5115ce7eba6f08ce9785d0c5",
                    "name": "OCCAMS-GB.DO50328.SA528818.wgs.20200402.aln.cram.crai"
                  }
                ],
                "inputForRuns": [
                  {
                    "runId": "wes-7ece63eab03c4d848d4a1a54908937e9"
                  }
                ]
              },
```
3. check song to see the at `BOTH` are published 
```
curl --location --request GET 'https://song.rdpc.cancercollaboratory.org/studies/OCCAMS-GB/analysis/6939ef36-4f0f-4a43-b9ef-364f0f0a4369' \

curl --location --request GET 'https://song.rdpc.cancercollaboratory.org/studies/OCCAMS-GB/analysis/2ab07a29-e008-4fd6-b07a-29e008bfd642' \
```

4. check platform and see one analysis missing
![image](https://user-images.githubusercontent.com/15824252/95253543-86ca1900-07ec-11eb-9fd7-72bc0f353909.png)

After discussion with @blabadi , 
- logs were not found in maestro for any indexing past September 25, 2020 
- anything published after September 25 is missing 

## Expected behaviour
Anything published in song should index into staging

**Comment by blabadi** [comment]
problem in kafka client had too many messages queued and didn't commit the poll since it pulled too many messages (500) at once, which after 5 minutes of working and not comitting to kafka results in reassigning the partition to the same consumer from last comitted offset. 

changed the max records per poll to 10 down from 500: 
![image.png](https://images.zenhubusercontent.com/5c99083bd2fde800015cf9fa/b00f5e1c-f07e-4cf6-8026-9af2d2c30c91)
I also lowered the attempts before moving msg to dlq to 10 instead of indefinite, to avoid clogging the consumer 

I'm not sure what was the reason kship got overwhelmed, could be the high number of retries or because it wasn't working for a while before since the network policy wasn't correct, but it worked when we fixed the network policy so I suspect something on the 17th of Sept happened.

a protective measure should be monitoring the offset of the kship and the offset of the latest message and make sure kship is not stuck and there is no big gap or some sort of monitoring of topic consumption not being halted.
